components/search/MoreFacetsDropdownOptions.vue

```diff
@@ -19,7 +19,13 @@
         :data-qa="`${filter.label} checkbox`"
         class="mb-3"
       >
+        <ColourSwatch
+          v-if="isColourPalette"
+          :hex-code="filter.label"
+          :aria-labelledby="`facet-field-${name}-${index}`"
```

Comment:
We now have this repeated four times, and end up with duplication of IDs in the two groups of fields (first nine and remainder).

Let's therefore apply these changes:
```diff
diff --git a/components/search/MoreFacetsDropdownOptions.vue b/components/search/MoreFacetsDropdownOptions.vue
index 0954148..1ca2214 100644
--- a/components/search/MoreFacetsDropdownOptions.vue
+++ b/components/search/MoreFacetsDropdownOptions.vue
@@ -22,10 +22,10 @@
         <ColourSwatch
           v-if="isColourPalette"
           :hex-code="filter.label"
-          :aria-labelledby="`facet-field-${name}-${index}`"
+          :aria-labelledby="fieldLabelId(index)"
         />
         <FacetFieldLabel
-          :id="`facet-field-${name}-${index}`"
+          :id="fieldLabelId(index)"
           :facet-name="name"
           :field-value="filter.label"
         />
@@ -48,10 +48,10 @@
           <ColourSwatch
             v-if="isColourPalette"
             :hex-code="filter.label"
-            :aria-labelledby="`facet-field-${name}-${index}`"
+            :aria-labelledby="fieldLabelId(index, { overLimit: true })"
           />
           <FacetFieldLabel
-            :id="`facet-field-${name}-${index}`"
+            :id="fieldLabelId(index, { overLimit: true })"
             :facet-name="name"
             :field-value="filter.label"
           />
@@ -124,6 +124,11 @@
     },
 
     methods: {
+      fieldLabelId(index, options = {}) {
+        const fieldIndex = options.overLimit ? index + this.limitTo : index;
+        return `facet-field-${this.name}-${fieldIndex}`;
+      },
+
       selectedHandler(value) {
         this.$emit('selectedOptions', this.name, value);
       }
```